addons/mail/static/src/core/common/attachment_upload_service.js

```diff
@@ -95,7 +95,10 @@ export class AttachmentUploadService {
                     }
                 }
                 const def = this.deferredByAttachmentId.get(tmpId);
-                this.unlink(this.store.Attachment.get(tmpId));
+                const tmpAttachment = this.store.Attachment.get(tmpId);
+                if (tmpAttachment) {
```

Comment:
still broken?

Reply:
Haven't checked this is still necessary or not. However considering that this code is in `bus.addEventListener` of `FILE_UPLOAD_LOADED`, I don't think there's any guarantee that it's called synchronously from another `bus.addEventListener` (`FILE_UPLOAD_ADDED`) adding the temp file.

So conceptually not guarding it was most certainly a mistake and prone to programming error crash.